spec/forms/candidate_interface/feedback_form_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::FeedbackForm, type: :model do
+  describe '#save' do
+    let(:application_form) { create(:application_form) }
+
+    it 'updates the application form with the satisfaction level and feedback if the form is valid' do
+      feedback_form = described_class.new(satisfaction_level: 'very_satisfied', suggestions: 'example')
+      feedback_form.save(application_form)
+
+      expect(application_form.feedback_satisfaction_level).to eq('very_satisfied')
+      expect(application_form.feedback_suggestions).to eq('example')
+    end
+
+    it 'accepts no satisfaction level or feedback' do
```

Comment:
Technically correct, but the wording is not very clear. Maybe something like they are optional or can have nil values. 🤷